modules/Individual Needs/moduleFunctions.php

```diff
@@ -134,3 +134,79 @@ function printINStatusTable($connection2, $guid, $gibbonPersonID, $mode = '', $a
 
     return $output;
 }
+
+function getInvestigationCriteriaArray($strand) {
```

Comment:
Will these values always be hard-coded, or maybe eventually a setting?

Reply:
Good question. I think they could become a setting longer term, but I thought, in the short term, where we might want to edit the whole thing in ways we can't anticipate, that hard-coding might make things easier. These questions have been refined at ICHK over time, and so should be generally useful, but I can imagine people wanting to customise.